### PR TITLE
Constant node

### PR DESCRIPTION
**Pull request by okdshin**
`Constant` is available on composite_backend.
And tensor attribute is available.

### CODE REVIEW

File: menoh/composite_backend/backend/generic/operator/constant.hpp [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+                          std::vector<array> const& input_list,
+                          std::vector<array> const& output_list) {
+                assert(input_list.size() == 0);
+                assert(output_list.size() == 1);
+                array value = attribute_tensor(node, "value");
+                std::cout << total_size(value) << std::endl;
+                for(decltype(total_size(value)) i = 0; i < total_size(value);
+                    ++i) {
+                    std::cout << fat(value, i) << std::endl;
+                    fat(output_list[0], i) = fat(value, i);
```
Comment (msakai) [inline menoh/composite_backend/backend/generic/operator/constant.hpp]: `fat` only works for `float16` type.
How about just using `menoh_dtype_size(value.dtype(), &bytes)` (or internal equivalent one) and then `memcpy`ing `total_size(value)*bytes`?
I think it is more general and maybe faster (assuming highly optimzied `__builtin_memcpy` is used).

File: menoh/composite_backend/backend/generic/operator/constant.hpp [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+            make_constant(node const& node,
+                          std::vector<array> const& input_list,
+                          std::vector<array> const& output_list) {
+                assert(input_list.size() == 0);
+                assert(output_list.size() == 1);
+                array value = attribute_tensor(node, "value");
+                std::cout << total_size(value) << std::endl;
+                for(decltype(total_size(value)) i = 0; i < total_size(value);
+                    ++i) {
+                    std::cout << fat(value, i) << std::endl;
```
Comment (msakai) [inline menoh/composite_backend/backend/generic/operator/constant.hpp]: I think it's better to remove those debug prints before merging to `master`.

**Comment by kmorito** [comment]
Hi

If there is a Constant node, the exception occurs on [line 214 of mkldnn_context.cpp](https://github.com/pfnet-research/menoh/blob/constant_node/menoh/composite_backend/backend/mkldnn/mkldnn_context.cpp#L214)
, and is caught and processing proceeds.
Is this not a problem?

File: menoh/composite_backend/backend/generic/operator/constant.hpp (hunk 2 above)
Comment (okdshin) [reply menoh/composite_backend/backend/generic/operator/constant.hpp]: Yes. Thanks pointing out.

**Comment by okdshin** [comment]
Hi @kmorito . Thank you for your question.
`mkldnn_with_generic_fallback` backend is an alias of `composite_backend` and `Constant` node is implemented in generic_context. So it is not a problem.
https://github.com/pfnet-research/menoh/blob/88167529ccfd35c8c81244bb02ca28d3bbcceec5/menoh/composite_backend/backend/generic/generic_context.cpp#L9

File: menoh/composite_backend/backend/generic/operator/constant.hpp (hunk 1 above)
Comment (okdshin) [reply menoh/composite_backend/backend/generic/operator/constant.hpp]: I see.

**Comment by okdshin** [comment]
Thanks for reviewing!

**Pull request merged**